land_claims.civmap.json

```diff
@@ -154,6 +154,7 @@
 {"name":"Droplandia","color":"#808080","declutter":3,"id":"50d5c49e-477c-52ab-8373-f04235ba6da2","polygon":[[[-12378,-1046],[-12555,-1153],[-12716,-908],[-12490,-810]]]},
 {"name":"(Joint Coalition Control)","shortname":"(JCC)","color":"#bd9c16","declutter":3,"x":-9160,"z":-2160,"id":"7fe5ad0e-c448-4a8e-b8b4-2af893b791cf","polygon":[[[-8834,-1983],[-8959,-1989],[-9043,-2001],[-9127,-2055],[-9168,-2058],[-9241,-2092],[-9294,-2079],[-9446,-2111],[-9491,-2175],[-9459,-2310],[-9456,-2384],[-9427,-2395],[-9448,-2421],[-9460,-2445],[-9446,-2483],[-9361,-2527],[-9285,-2473],[-9302,-2437],[-9310,-2368],[-9315,-2343],[-9253,-2288],[-9229,-2269],[-9149,-2278],[-9086,-2272],[-9026,-2251],[-8900,-2218],[-8836,-2252],[-8783,-2193],[-8801,-2110],[-8794,-2036],[-8833,-1981]]]},
 {"name":"(Joint Coalition Control)","shortname":"(JCC)","color":"#bd9c16","declutter":3,"x":-7100,"z":400,"id":"6b5de567-c06f-4951-8cc6-9a18734dadc2","polygon":[[[-7419,474],[-7434,438],[-7433,381],[-7393,335],[-7366,317],[-7306,305],[-7243,261],[-7223,214],[-7195,189],[-7180,147],[-7165,131],[-7137,153],[-7106,185],[-7079,217],[-7068,254],[-7042,255],[-7021,276],[-7008,309],[-7005,339],[-6981,351],[-6966,369],[-6932,387],[-6915,420],[-6887,437],[-6868,453],[-6824,462],[-6774,517],[-6827,586],[-6885,610],[-6921,607],[-6944,611],[-6972,642],[-7000,650],[-7022,691],[-7043,703],[-7052,695],[-7060,694],[-7065,701],[-7071,708],[-7091,722],[-7109,736],[-7124,748]]]},
+{"id":"1dde50a4-27af-4d49-9ae8-86cc58f68ef6","collectionId":"civmap:collection/user","polygon":[[[-1444,-4641],[-1503,-4685],[-1542,-4651],[-1611,-4647],[-1639,-4675],[-1645,-4695],[-1659,-4717],[-1709,-4744],[-1781,-4838],[-1781,-4884],[-1716,-4915],[-1644,-4979],[-1418,-5124],[-1272,-4970],[-1190,-4886],[-1142,-4770],[-1204,-4694],[-1281,-4635]]],"name":"Jolington"}
```

Comment:
add a , at the end, remove the collectionId entry, move the name entry to the beginning, add a color, set declutter to like 3 I guess, and add yourself as contact, please.